xwrf/postprocess.py

```diff
@@ -24,6 +24,14 @@ def _decode_times(ds: xr.Dataset) -> xr.Dataset:
         )
     ds = ds.assign_coords({'Time': _time})
     ds.Time.attrs = {'long_name': 'Time', 'standard_name': 'time'}
+    # make XTIME be consistent with its description
+    if 'XTIME' in ds.variables and np.issubdtype(ds.XTIME.dtype, np.datetime64):
+        ds['XTIME'].data = (
+            ds.XTIME.data
+            - pd.to_datetime(
+                ds['XTIME'].description, format='minutes since %Y-%m-%d %H:%M:%S'
+            ).to_datetime64()
```

Comment:
Actually I'm confused. The  title  says `timedelta` but theoutput here is `datetime`.  

Would it work to pass `decode_timedelta` to`open_dataset`?

Reply:
It is supposed to create a `timedelta` from the difference between `ds.XTIME.data` (dtype `datetime64`) and `pd.to_datetime.to_datetime64()` :)

Reply:
Ah missed that with the funny formatting. Thanks!